src/components/app/list-new/AppList.tsx

```diff
@@ -294,6 +294,12 @@ const AppList = ({ isArgoInstalled }: AppListPropType) => {
             />
         )
 
+    const removePageNumber = (search: string) => {
+        const searchParams = new URLSearchParams(search)
+        searchParams.delete('pageNumber')
```

Comment:
Ahould be ideally used from fe-common after exporting URL_FILTER_KEYS
```suggestion
        searchParams.delete(URL_FILTER_KEYS.PAGE_NUMBER)
```

Reply:
will pick later.